core/inflation/src/lib.rs

```diff
@@ -75,6 +76,20 @@ fn inflate(x: Balance, years: u8) -> Option<Balance> {
 /// Reference(s):
 /// - <https://github.com/evolutionlandorg/bank/blob/master/contracts/GringottsBank.sol#L280>
 pub fn deposit_interest(amount: Balance, months: u8) -> Balance {
+	// The result of `((quot - 1) * precision + rem * precision / d)` is `197` when months is
+	// `12`.
+	//
+	// The default interest is `1_000`.
+	// So, we directly use `1_970_000` here instead `interest * 197 * 10^7`.
+	fn f(amount: U256, precision: U256, quot: U256, rem: U256, d: U256) -> Option<Balance> {
+		Some(
+			(amount.checked_mul(
+				precision.checked_mul(quot.checked_sub(1_u8.into())?)? + precision * rem / d,
```

Comment:
`precision * rem` : precision is a fixed U256 value and rem is a U256 value, comes from n and d. Does it necessary to use checked_mul here?

Reply:
`quot.checked_sub(1_u8.into())` is a var.